### PR TITLE
ProductVersion model

### PR DESCRIPTION
**Pull request by lioramilbaum**
Product Version is the primary filter for container automation data
/cc @mzidek-gh

### CODE REVIEW

File: estuary/models/productversion.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from neomodel import StringProperty
+
+from estuary.models.base import EstuaryStructuredNode
+
+
+class ProductVersion(EstuaryStructuredNode):
+    """Definition of RHEL Product Version in Neo4j."""
+
+    name = StringProperty()
```
Comment (mprahl) [inline estuary/models/productversion.py:9]:
```suggestion
    name = StringProperty(unique=True, index=True)
```

File: tests/functional/test_ProductVersion.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import json
+
+import pytest
+import requests
+from estuary.models.productversion import ProductVersion
```
Comment (mprahl) [inline tests/functional/test_ProductVersion.py:5]: I think isort (#297) would change this.

Comment (lioramilbaum) [reply tests/functional/test_ProductVersion.py:6]: fixed

File: estuary/models/productversion.py (hunk 1 above)
Comment (lioramilbaum) [reply estuary/models/productversion.py]: According to neomodel documentation, unique_index argument is the same as using unique and index arguments. WDYT?

Comment (mprahl) [reply estuary/models/productversion.py]: Sounds good!

**Review by mprahl — APPROVED** [review]

File: estuary/models/productversion.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from neomodel import StringProperty
+
+from estuary.models.base import EstuaryStructuredNode
+
+
+class ProductVersion(EstuaryStructuredNode):
+    """Definition of RHEL Product Version in Neo4j."""
```
Comment (mprahl) [inline estuary/models/productversion.py:7]: Sorry, I missed this before. I think it'd be best if this was just a generic concept of a product version.
```suggestion
    """Definition of Product Version in Neo4j."""
```

**Pull request merged**